frontend/src/styles.css

```diff
@@ -0,0 +1,58 @@
+/* Global styles */
+.uppercase {
+  letter-spacing: 0.06em;
+}
+
+/* Regex Colorizer no bg theme */
+/* https://stevenlevithan.com/regex/colorizer/themes/nobg.css */
+
+.regex {
+  color: var(--sl-color-neutral-500);
+}
+/* metasequence */
+.regex b {
+  color: var(--sl-color-blue-500);
+}
+/* char class */
+.regex i {
+  color: var(--sl-color-lime-500);
```

Comment:
FYI @Shrinks99 the regex colorizer theme uses additional colors that map nicely to [Shoelace color tokens](https://shoelace.style/tokens/color), but are missing from GH primer primitives (lime, amber, rose, teal, fuchsia.) It'd be nice to still support these additional colors as we need them, thoughts on keeping the Shoelace theme colors around?

Reply:
The Tailwind palette is quite large (which is great for Tailwind because it's supposed to adapt to your project) almost to the point where I feel kind of overwhelmed when designing delineating between hues like orange vs amber and when each should be used.  I like Primer's reduced palette for this reason as the reduction in flexibility comes with stronger associations as each hue and its corresponding lightness values are (hopefully) used with purpose.

I don't think we should keep both because of the incongruities between palettes... But do agree that for things like syntax highlighting where more hues appear to be required it can get a little tough.  Defining a specific palette perhaps _based_ on Primer's colors to be used for syntax highlighting could be one approach?

Reply:
The Shoelace palette is actually what's implemented right now--the Tailwind palette [uses shoelace colors](https://github.com/webrecorder/browsertrix-cloud/blob/main/frontend/tailwind.config.js#L8) heh this could also be documented in Storybook.